cmd/manager/operator.go

```diff
@@ -489,3 +495,39 @@ func serveCRMetrics(cfg *rest.Config, operatorNs string) error {
 	}
 	return nil
 }
+
+// createNonComplianceAlert tries to create the default PrometheusRule. Returns nil.
+func createNonComplianceAlert(ctx context.Context, client *monclientv1.MonitoringV1Client, namespace string) error {
+	rule := monitoring.Rule{
+		Alert: "NonCompliant",
+		Expr:  intstr.FromString(`compliance_operator_compliance_state{name=~".+"} == 1`),
```

Comment:
This doesn't alert on an inconsistent state nor an error in the compliance scan. We should adjust accordingly.

Reply:
Ah, I wasn't sure if this would have needed separate rules for the possible states. I made it > 0 to trigger on any of the non-compliant states.